test/hotspot/jtreg/vmTestbase/nsk/jvmti/SetLocalVariable/setlocal001/setlocal001.cpp

```diff
@@ -74,23 +75,23 @@ void JNICALL Breakpoint(jvmtiEnv *jvmti_env, JNIEnv *env,
             if (strcmp(table[i].name, "l") == 0) {
                 err = jvmti_env->SetLocalLong(thr, 1,
                     table[i].slot, longVal);
-                if (err != JVMTI_ERROR_NONE) {
+                if (err != JVMTI_ERROR_NONE && !(is_virtual && err == JVMTI_ERROR_OPAQUE_FRAME)) {
```

Comment:
Maybe add a new API called something `invalidError(err)`. Not necessary, but might be a little better.

Reply:
Thanks. I was also thinking about this but was not sure it is worth it.
Will update now as there are two of us now. :)